pkg/generators/models/testdata/cases/const/api.yaml

```diff
@@ -0,0 +1,11 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+components:
+  schemas:
+    Foo:
```

Comment:
Why new tests cases, there are already cases for 

`contants` https://github.com/contiamo/openapi-generator-go/tree/master/pkg/generators/models/testdata/cases/constants

and `enums` https://github.com/contiamo/openapi-generator-go/tree/master/pkg/generators/models/testdata/cases/enums